### DISCUSSION TITLE
Localization RTL

### DISCUSSION THREAD

**Discussion by coopatek** [oqtane/oqtane.framework#3481] [Q&A]
when setting the language as Arabic, the display should be converted to RTL but itis not, is there anything I am missing ?

**Comment by sbwalker** [comment]
@coopatek the current default theme does not include RTL support

**Comment by hishamco** [comment]
I might work on this if there's no objections

**Comment by sbwalker** [comment]
@hishamco since you are very familiar with this concept, please suggest the approach you would take to add RTL support to the framework. This will help everyone understand the scope of the changes required and ensure they align with the architecture/goals before we actually implement them.

**Reply by hishamco** [reply]
There are two approaches for RTL:

1. Create a new style sheet that takes care of RTL languages including margin, padding, and other related properties
2. Generates an RTL support in the build pipeline using Glup

IMHO we could for the 1st approach at the begging because it's simple and it will not take much effort, then we can go to auto-generation as long run technique because it will not require any manual changes if the LTR styles have been changed
